project/models.py

```diff
@@ -41,6 +41,8 @@ class Issue(models.Model):
     FREE, EASY, MEDIUM, HARD, VERY_EASY = 0, 1, 2, 3, 4
     FREE_READ, VERY_EASY_READ, EASY_READ, MEDIUM_READ, HARD_READ = "Free", "Very-Easy", "Easy", "Medium", "Hard" \
         # Human Readable Names
+    FREE_COLOR, VERY_EASY_COLOR, EASY_COLOR, MEDIUM_COLOR, HARD_COLOR = "#f2f2f2", "", "#00b74a", "#ffa900", "#f93154" \
```

Comment:
Very_easy_color seems to be empty, even if it is set by default add the color code